### ISSUE TITLE
Set default zoom level works but logs an error to console on LT startup

### ISSUE THREAD

**Issue by ackalker** [LightTable/LightTable#1343]
I'm using the "App: Set the default zoom level" user behaviour (thanks for implementing this, btw :) ) and it works fine, but it logs an error to LT's console on startup:

```
"Invalid behavior: :lt.objs.app/set-default-zoom-level"
Error: Unable to convert 'args' passed to CallObjectMethodSync
    at Window.eval [as zoomLevel] (window_bindings.js:199:3)
    at Function.lt.objs.app.__BEH__set_default_zoom_level (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/node_modules/lighttable/bootstrap.js:21251:36)
    at c (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/node_modules/lighttable/bootstrap.js:6196:14)
    at a (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/node_modules/lighttable/bootstrap.js:6236:18)
    at lt.object.raise_STAR_ (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/node_modules/lighttable/bootstrap.js:20529:74)
    at a (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/node_modules/lighttable/bootstrap.js:20562:34)
    at b (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/node_modules/lighttable/bootstrap.js:20566:14)
    at Object.lt.objs.app.init (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/node_modules/lighttable/bootstrap.js:21117:19)
    at HTMLScriptElement.script.onload (file:///home/miki/vcs/git/LightTable/LightTable/deploy/core/LightTable.html:43:24)
```

This is because of a small bug in node-webkit which I've reported as: 
https://github.com/rogerwang/node-webkit/pull/1395 and https://github.com/rogerwang/node-webkit/issues/1520 , and which has since been fixed.

As I'm not sure which (customized?) node-webkit LT uses, I'm reporting it here.
Please update LT's node-webkit to at least version 0.9.0 (Git tag nw-0.9.0) to get this fix.

Thanks in advance.

**Comment by binary132** [comment]
I'm still getting this issue on 0.7.2 downloaded as a .tar.gz from the main site.

**Comment by kenny-evitt** [comment]
@binary132 Try building the latest source version and comment whether you can reproduce this issue.

**Comment by binary132** [comment]
Can't reproduce bug when building current master (0.8.0).  I thought the fix was in for 0.7.2 for some reason though.
